src/plugins/intel_cpu/src/cpu_map_scheduling.cpp

```diff
@@ -90,7 +91,7 @@ bool get_cpu_pinning(bool& input_value,
             if ((streams_info_table[0][PROC_TYPE] == ALL_PROC) &&
                 (streams_info_table[1][PROC_TYPE] != EFFICIENT_CORE_PROC) &&
                 (streams_info_table[2][PROC_TYPE] == EFFICIENT_CORE_PROC)) {
-                result_value = false;
+                result_value = cpu_reservation;
```

Comment:
Please update login in windows as well. When user only set reservation to true, update pinning to true as well. If user set pinning together, keep user setting.

Reply:
Ok, done. Please take a look.

Reply:
Why cannot we update cpu_pinning value inside config.cpp? It sounds logical to describe this trict dependency between properties directly in the config

Reply:
Now cpu pinning cannot support the stream using both Pcore and Ecore. 

We update default CPU pinning value here because the dependencies of platform and stream information.